packages/core/src/components/advanced-select/advanced-select.tsx

```diff
@@ -161,7 +161,12 @@ export class AdvancedSelect implements ComponentInterface {
                       "options" in option && "summaryCounter" in option && !!option?.summaryCounter,
                   )
                   .map((group) => {
-                    return <dso-badge status={group.variant ?? "outline"}>{group.options.length}</dso-badge>;
+                    return (
+                      group.options &&
+                      group.options.length && (
```

Comment:
Kan dit niet gecombineerd worden tot `group.options?.length` ? En functioneel wijzigt er nu ook iets: Vroeger kon er met `summaryCounter` een 0 in de badge staan, nu niet meer: is dat de bedoeling?

Verder heb ik mijn twijfels of dit in deze `map` moet zitten: De return type van deze map is nu ook `undefined` (en `number`) geworden. Zou je de `filter()` op regel 159 en de `map()` op regel 163 niet beter tot 'n reduce kunnen combineren?